cmd/soroban-cli/src/commands/config/mod.rs

```diff
@@ -49,6 +60,42 @@ impl Args {
         Ok(key.key_pair(self.hd_path)?)
     }
 
+    pub async fn sign_with_local_key(&self, tx: Transaction) -> Result<TransactionEnvelope, Error> {
+        let signer = LocalKey::new(self.key_pair()?, false);
```

Comment:
Everywhere I see local key used the prompt is set to false. Do we use the prompting at all, or is that dead code in this change?